### ISSUE TITLE
Vytvořit carousell na banner

### ISSUE THREAD

**Issue by LordShedy** [LordShedy/unidens.com#10]
### Popis
Je třeba mít carousell na banner. Ideálně s co nejméně ECS (Javascript).

**Comment by biel00** [comment]
### Carousel technológia
Dnes na obede mi Arturas spomínal [Bootstrap](https://getbootstrap.com/), ktorý by možno stál za vyskúšanie.

**Comment by LordShedy** [comment]
Ano, s Bootstrapem je to velmi jednoduché, ale je tu problém, že když se ho rozhodneš používat, že ho musíš používat napříč **celým** webem a to by znamenalo, že musíme používat bootstrapí CSS a ne klasická. Na jednu stranu ano, bylo by to rychlejší a jednodušší, ale nevýhoda by byla, že bychom měli poměrně unifikovaný design, který se nedá moc (snadno) upravovat a že bychom byli omezení co se schopností boostrapu týče. Vanila CSS je pracnější, ale dává to plně volnou ruku. Nejsem vyloženě proti bootstrapu, ale preferoval bych vanilla CSS.

**Comment by biel00** [comment]
Chápem kam tým mieriš, a musím ti dať za pravdu. Takže pôjdeme po vanilla javascripte, css a html, aby sme mali plnú kontrolu nad celým kódom stránky. Za mňa ok. Ak súhlasíš aj ty, prosím uzavri Issue.

**Comment by LordShedy** [comment]
Ok, tenhle issue uzavřeme, až bude carousell dokončený a nasazený.
